### DISCUSSION TITLE
App Suggestion emacs with ess

### DISCUSSION THREAD

**Discussion by trdunsworth** [dat-linux/community#20] [App, Extra, Book suggestions]
Would it be possible to add emacs with the ESS plugin enabled? This would be great for non-Vim users.

**Comment by dat-linux** [comment]
Hi, thanks for the suggestion. It's definitely possible. Stay tuned.

**Comment by dat-linux** [comment]
Emacs+ESS added to the control panel and extras.

Please "sudo apt update && sudo apt upgrade" or "Menu->Preferences->Apply full upgrade" to upgrade.

**Reply by trdunsworth** [reply]
Thank you very much for that!
